### ISSUE TITLE
Default plugins take too long to load

### ISSUE THREAD

**Issue by pakrym** [Azure/autorest#3339]
Removing the following plugins from the default list cuts 3 second off the autorest load time greatly speeding up the inner dev loop.

```
  - $(this-folder)plugin-powershell.md
  - $(this-folder)plugin-azureresourceschema.md
  - $(this-folder)plugin-java.md
  - $(this-folder)plugin-nodejs.md
  - $(this-folder)plugin-php.md
  - $(this-folder)plugin-python.md
  - $(this-folder)plugin-ruby.md
  - $(this-folder)plugin-typescript.md
```

But even 3 seconds seems like a lot to load couple configuration files.

**Comment by fearthecowboy** [comment]
Yes; aware of this. 

will be looking into this soonish.

**Comment by fearthecowboy** [comment]
fixed in autorest 3.0.6180+
